plugins/wm-actions/wm-actions.cpp

```diff
@@ -203,7 +207,8 @@ class wayfire_wm_actions_t : public wf::plugin_interface_t
     {
         auto view = choose_view(ev.source);
 
-        return toggle_keep_above(view);
+        in_always_above_layer = !in_always_above_layer;
```

Comment:
in_always_above_layer is a per-output variable, but we want to track this state per-view, right? There can be multiple always-on-top views (so we cannot use a single boolean variable to keep track of their state).

There is in fact no real need for the `in_always_above_layer` at all (I think), we store custom_data on each view in the above layer, so the state of a view can be checked via `view->has_data(...)`